kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -151,117 +156,109 @@ function redirectToExploreChannel(store) {
   store.dispatch('CORE_SET_PAGE_LOADING', true);
   store.dispatch('SET_PAGE_NAME', PageNames.EXPLORE_ROOT);
 
-  _getCurrentChannelId()
-    .then((currentChannelId) => {
-      store.dispatch('CORE_SET_ERROR', null);
-      if (currentChannelId) {
-        store.dispatch('SET_CURRENT_CHANNEL', currentChannelId);
-        cookiejs.set('currentChannel', currentChannelId);
+  ChannelResource.getCollection({}).fetch().then(
+    (channelsData) => {
+      const channelList = _channelListState(channelsData);
+      const channelId = _getDefaultChannelId(store, channelList);
+      _handleChannels(store, channelId, channelList);
```

Comment:
For example, this will throw an error because the channel is not found, but then the router will carry on and try to execute the action to set the page state based on this channelId.